HALF_OF multiplies by a double 0.5

When compiling with msvc 2013 I get this warning:

> warning C4244: '=' : conversion from 'double' to 'float', possible loss of data

It looks like a bright mind figured this out before I did. credits to him!
https://github.com/steffen-kiess/kissfft/commit/a10e9c481c7faf542004be501cd663129be9dff3

--- a/_kiss_fft_guts.h
+++ b/_kiss_fft_guts.h
@@ -135,7 +135,7 @@ struct kiss_fft_state{
 #else
 #  define KISS_FFT_COS(phase) (kiss_fft_scalar) cos(phase)
 #  define KISS_FFT_SIN(phase) (kiss_fft_scalar) sin(phase)
-#  define HALF_OF(x) ((x)*.5)
+#  define HALF_OF(x) ((x)*((kiss_fft_scalar).5))
 #endif
 
 #define  kf_cexp(x,phase) \

